Sources/Sentry/SentryNSDictionarySanitize.m

```diff
@@ -8,6 +8,10 @@
         return nil;
     }
 
+    if ([dictionary isEqual:[NSNull null]]) {
```

Comment:
`m`: It's a small edge case, but we could strengthen this check by instead asking:
```suggestion
    if (![[dictionary class] isSubclassOfClass:[NSDictionary class]]) {
```
that way if for whatever reason any other pointer comes in here inadvertently, we don't wind up with an unrecognized selector in the messages sent to it below.

Reply:
That is a good addition! Thanks for that.